src/js/jsx/shared/TitleHeader.jsx

```diff
@@ -38,6 +39,12 @@ define(function (require, exports, module) {
         render: function () {
             var workingTitle = this.props.title;
 
+            var classNameSet = {
+                "section-title": true
+            };
+
+            var className = classnames(classNameSet, this.props.className);
```

Comment:
you can merge these lines into one line `var className = classnames("section-title", this.props.className)`